caddyconfig/caddyfile/parse.go

```diff
@@ -212,7 +212,7 @@ func (p *parser) addresses() error {
 
 	for {
 		tkn := p.Val()
-
+		token := p.Token()
```

Comment:
I think we should rename `tkn` to "value" or something, just to differentiate it from `token`. Kinda confusing that they have different types with essentially the same name.